framework/console/widgets/Table.php

```diff
@@ -69,15 +69,15 @@ class Table extends Widget
     /**
      * @var array table headers
      */
-    private $_headers = [];
+    protected $_headers = [];
```

Comment:
protected variables should not start with `_`

Reply:
So just removing the `_` everywhere would be fine?

Reply:
> protected variables should not start with `_`

`_` prefix is for avoiding conflicts between regular properties and virtual properties from getters and setters. Visibility should not matter here (this would be against PSR-2).